openmdao/utils/general_utils.py

```diff
@@ -506,14 +506,16 @@ def pad_name(name, pad_num=10, quotes=False):
             return '{0}'.format(name)
 
 
-def run_model(prob):
+def run_model(prob, allow_exception=False):
```

Comment:
would 'ignore_exception' be clearer here?

Reply:
Changed to `ignore_exception`